pages/_app.tsx

```diff
@@ -19,7 +20,7 @@ if (process.env.NEXT_PUBLIC_API_MOCKING === 'enabled') {
   require('../src/mocks')
 }
 
-function MyApp({ Component, pageProps }: AppProps) {
+export default function MyApp({ Component, pageProps }: AppProps) {
```

Comment:
rename MyApp to WalloraApp. MyApp is given in sample code of getting started tutorial due to lack of a proper app name